contents/javascript/Javascript-framework-Angular.md

```diff
@@ -30,12 +30,22 @@ Consider a simple web application that displays the name of a person as shown be
     The sandbox above is editable. You can enter your name in the text box and see how the UI is automatically updated. You can also view and edit the code by clicking on the Both button. 
 </box>
 
-The UI of an Angular app is defined in _templates_ which contain a mix of HTML code and HTML-like custom tags.
+The UI of an Angular app is defined in _templates_ which contain a mix of HTML code and HTML-like custom tags. The template for the above example is given below.
+
+``` html
+<h1>About Me!</h1>
+<h2>Hello World! My name is {hero.name}</h2>
+<div>
+<label>NAME:
+  <input [(ngModel)]="hero.name"/>
+</label>
+</div>
+```
```

Comment:
The UI (as seen by the viewer) has no mention of heros. Why does the code use the term `hero`?

Reply:
The component file for this template has a variable `hero` as follows:

```typescript
export class HeroesComponent {
  hero: Hero = {
    name: 'John Doe'
  };
}
```
so the hero refers to this variable

Reply:
The question in the reader's mind is why use that unusual term at all? Based on what is seen on the web page, there is no reason to use that term. If the page heading is `Hero Profile` or something like that, at least there is some reason to use that term.

Reply:
Should I rename the variable to `AboutMe`? then it can be referenced as `AboutMe.name`

Reply:
`person`? `person.name` feels more meaningful.

Reply:
Sure will do that